smart_error.gemspec

```
@@ -29,4 +29,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.6'
   spec.add_dependency 'activesupport', '>= 5.0'
   spec.add_dependency 'i18n', '>= 0.8'
+  spec.add_dependency 'rubocop'
+  spec.add_dependency 'rubocop-performance'
```

Those are development dependency, not required for users
```suggestion
  spec.add_development_dependency 'rubocop'
  spec.add_development_dependency 'rubocop-performance'
```